docker-compose.yml

```diff
@@ -131,6 +131,33 @@ services:
       # apply middleware to route
       - "traefik.http.routers.functions-app.middlewares=fn-app-stripprefix"
 
+  functions-app-messages:
+    image: io-functions-app-messages
+    build:
+      context: ${FUNCTIONS_APP_MESSAGES_CONTEXT}
+      dockerfile: ./Dockerfile
+      args:
+        - FUNCTIONS_APP_MESSAGES_COMMIT_SHA
+    env_file:
+      - generated-env/env.cosmosdb
+      - generated-env/env.io-functions-app-messages
+    working_dir: /usr/src/app
+    ports:
+      - ${FUNCTIONS_APP_MESSAGES_PORT}:7071
+    networks:
+      - io-fn
+    depends_on:
+      - functions-node-14
+      - fixtures
+      - traefik
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.functions-app-messages.rule=Host(`functions-app-messages.localhost`) || PathPrefix(`/functions-app-messages`)"
+      - "traefik.http.routers.functions-app-messages.entrypoints=websecure"
+      - "traefik.http.routers.functions-app-messages.tls=true"
+      # apply middleware to route
+      - "traefik.http.routers.functions-app-messages.middlewares=fn-app-messages-stripprefix"
```

Comment:
is needed even the definition of the strip-prefix middleware in the traefik labels
```
- "traefik.http.middlewares.fn-app-messages-stripprefix.stripprefix.prefixes=/functions-app-messages"
```